### PR TITLE
Increase CLI test timeout/Restore core image version

### PR DESCRIPTION
**Pull request by baum**
1. Increase CLI test timeout
2. Restore core image version

Signed-off-by: Alexander Indenbaum <aindenba@redhat.com>

### CODE REVIEW

**Review by liranmauda — COMMENTED** [review]
Why do we need 10 min timeout (instead of 3 which is big on its own)?

**Comment by dannyzaken** [comment]
@liranmauda I agree that this is a long timeout. for some reason, with the newer images of noobaa-core 5.9 the tests fail on timeout. 
we still need to find the root cause but for now, we just want to use the newer image of 5.9 without failing the operator tests.
I will open an issue on that

**Review by dannyzaken — APPROVED** [review]

File: pkg/options/options.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ const (
 	// ContainerImageRepo is the repo of the default image url
 	ContainerImageRepo = "noobaa-core"
 	// ContainerImageTag is the tag of the default image url
-	ContainerImageTag = "5.9.0-20210722"
+	ContainerImageTag = "5.9.0-20211118"
```
Comment (dannyzaken) [inline pkg/options/options.go:33]: @liranmauda why is this image tag in the future? 🤔

Comment (liranmauda) [reply pkg/options/options.go:33]: Being a good developer means using future, unwritten, undeveloped builds 😂 

Anyway, it was supposed to be 20211018, you can change that to the latest...

**Comment by dannyzaken** [comment]
I updated the core image in this PR https://github.com/noobaa/noobaa-operator/pull/824

**Pull request closed without merging**